Formula/texlive.rb

```diff
@@ -0,0 +1,499 @@
+    inreplace buildpath/"texk/kpathsea/texmf.cnf",
+      "TEXMFROOT = $SELFAUTOPARENT", "TEXMFROOT = $SELFAUTODIR/share"
+    inreplace share/"texmf-dist/web2c/texmfcnf.lua",
+      "selfautoparent:texmf", "selfautodir:share/texmf"
+
+    # Fix path resolution in some scripts (these should be upstreamed)
+    pathfix_files = [
+      buildpath/"texk/texlive/linked_scripts/cjk-gs-integrate/cjk-gs-integrate.pl",
+      buildpath/"texk/texlive/linked_scripts/crossrefware/bbl2bib.pl",
+      buildpath/"texk/texlive/linked_scripts/crossrefware/bibdoiadd.pl",
+      buildpath/"texk/texlive/linked_scripts/crossrefware/bibmradd.pl",
+      buildpath/"texk/texlive/linked_scripts/crossrefware/biburl2doi.pl",
+      buildpath/"texk/texlive/linked_scripts/crossrefware/bibzbladd.pl",
+      buildpath/"texk/texlive/linked_scripts/crossrefware/ltx2crossrefxml.pl",
+      buildpath/"texk/texlive/linked_scripts/texlive/tlmgr.pl",
+      buildpath/"texk/texlive/linked_scripts/pedigree-perl/pedigree.pl",
+      buildpath/"texk/texlive/linked_scripts/tlshell/tlshell.tcl",
+      share/"tlpkg/TeXLive/TLUtils.pm",
+    ]
+
+    inreplace pathfix_files, "SELFAUTOPARENT", "TEXMFROOT"
+
+    args = std_configure_args + [
+      "--disable-dvisvgm", # needs its own formula
+      "--disable-missing",
+      "--disable-native-texlive-build", # needed when doing a distro build
+      "--disable-ps2eps", # provided by ps2eps formula
+      "--disable-psutils", # provided by psutils formula
+      "--disable-static",
+      "--disable-t1utils", # provided by t1utils formula
+      "--enable-build-in-source-tree",
+      "--enable-shared",
+      "--enable-compiler-warnings=yes",
+      "--with-banner-add=/#{tap.user}",
+      "--with-system-clisp-runtime=system",
+      "--with-system-cairo",
+      "--with-system-freetype2",
+      "--with-system-gd",
+      "--with-system-gmp",
+      "--with-system-graphite2",
+      "--with-system-harfbuzz",
+      "--with-system-icu",
+      "--with-system-libpng",
+      "--with-system-mpfr",
+      "--with-system-ncurses",
+      "--with-system-pixman",
+      "--with-system-potrace",
+      "--with-system-zlib",
+    ]
+
+    args << if OS.mac?
+      "--without-x"
+    else
+      # Make sure xdvi uses xaw, even if motif is available
+      "--with-xdvi-x-toolkit=xaw"
+    end
+
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+    system "make", "texlinks"
+
+    # Create tlmgr config file.  This file limits the actions that the user
+    # can perform in 'system' mode, which would write to the cellar.  'tlmgr' should
+    # be used with --usermode whenever possible.
+    (share/"texmf-config/tlmgr/config").write <<~EOS
+      allowed-actions=candidates,check,dump-tlpdb,help,info,list,print-platform,print-platform-info,search,show,version,init-usertree
+    EOS
+
+    # Delete some binaries that are provided by existing formulae as newer versions.
```

Comment:
Can't we disable these?

Reply:
These are actually just Perl scripts that are linked into `bin` so there is no build process to disable.  I will adjust the comment since it incorrectly says these are binaries.